mod/invitefriends/start.php

```diff
@@ -36,20 +35,7 @@ function invitefriends_page_handler($page) {
 		return false;
```

Comment:
Can't see what's here on my phone. Move all logic out of page handler and move to resource view (gatekeeper or whatever that is)